src/commissaire/models/__init__.py

```diff
@@ -288,15 +293,17 @@ class Cluster(Model):
     _attribute_map = {
         'name': {'type': str},
         'status': {'type': str},
-        'type': {'type': str},
```

Comment:
We're still early enough that we can get away with these backward-compatibility breaks, right?

At some point we may have to worry about how to migrate data for changes like this.